Change exit code for CLI flag error (#7134)
stylelint/stylelint: docs/migration-guide/to-16.md

- 16.13.0
- 18.0.0

## Changed CLI exit code for flag error

> When there was some CLI flag error, the CLI newly exits with `64` instead of `2`. This change clarifies the exit code by rule problems.

Reviewer: ```suggestion
We've changed the exit code for CLI flag errors from `2` to `64` so that `2` is only used for lint problems. 
```

Author: Thanks. Fixed!
